editor.planx.uk/src/pages/FlowEditor/lib/analytics/types.ts

```diff
@@ -32,6 +32,8 @@ export type AllowListKey = (typeof ALLOW_LIST)[number];
  */
 export type HelpClickMetadata = Record<string, string>;
 
+export type SaveClickMetadata = Record<string, string>;
```

Comment:
This seems redundant and unused. There's no metadata to gather - it's just a true/false.

Other analytics event gather metadata in order to store additional data, but this is just a straightforward boolean.